arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json

```diff
@@ -2130,11 +2096,35 @@
         },
         "backupManagementType": {
           "description": "Type of backup managmenent for the job. Possible value: AzureIaasVM.",
-          "type": "string"
+          "enum": [
+            "Invalid",
+            "AzureIaasVM",
+            "MAB",
+            "DPM",
+            "AzureBackupServer",
+            "AzureSql"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "BackupManagementType",
+            "modelAsString": false
+          }
         },
         "operation": {
           "description": "Type of operation. Possible values: ConfigureBackup, Backup, Restore, Unprotect, DeleteBackupData.",
```

Comment:
You really don't need to have `possible values` listed, AutoRest will actually add that text when it generates SDKs.